### PR TITLE
Add comscore behind TCfv2

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?

Adds  comscore behind TCfv2 following the documentation details : http://images.go.comscore.com/Web/comScoreInc/%7B97ce31a6-0786-4bf6-bd82-c0d376dade6a%7D_IAB-Europe-TCF-v20-Measurement-Solution%e2%80%af.pdf?utm_campaign=EMEA_GB_JUL2020_ST_TCF_V20&utm_medium=email&utm_source=comscore_elq_EMEA_GB_JUL2020_ST_TCF_V20

Still pending to check if we want to use the same TCFV1 API for sending the consent to them

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### CODE REVIEW

**Comment by sndrs** [comment]
@mxdvl want to take a look over this? i've updated it to use our general CMP v4 check rather than just the test

File: static/src/javascripts/projects/commercial/modules/comscore.js [hunk 1]
```diff
@@ -14,6 +15,7 @@ type comscoreGlobals = {
 const comscoreSrc = '//sb.scorecardresearch.com/cs/6035250/beacon.js';
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/comscore.js:15]: As mentioned previously, this URL fails with a 404…

File: common/app/views/fragments/analytics/comscore.scala.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
 @if(ComscoreSwitch.isSwitchedOn) {
     <noscript>
         @defining(getContent(page).map(_.tags.keywords.map(_.name)).map(_.mkString(","))) { keywords =>
-            <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&comscorekw=@keywords.map({kw => URLEncoder.encode(kw, "UTF-8")})" />
+            <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&gdpr=0&comscorekw=@keywords.map({kw => URLEncoder.encode(kw, "UTF-8")})" />
```
Comment (mxdvl) [inline common/app/views/fragments/analytics/comscore.scala.html:9]: What does this do exactly?

**Review by mxdvl — APPROVED** [review]
👍 LGTM

File: static/src/javascripts/projects/commercial/modules/comscore.js (hunk 1 above)
Comment (sndrs) [reply static/src/javascripts/projects/commercial/modules/comscore.js:15]: yeah, this was present in the PR already ¯\_(ツ)_/¯

File: common/app/views/fragments/analytics/comscore.scala.html (hunk 2 above)
Comment (sndrs) [reply common/app/views/fragments/analytics/comscore.scala.html:9]: not sure, i've only updated the test checks

**Comment by mxdvl** [comment]
We actually need to only fire the script if consent is given.

**Pull request merged**